.github/workflows/ci.yml

```diff
@@ -49,3 +49,4 @@ jobs:
         id: emissary
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+          dry-run: 'true'
```

Comment:
could you make a couple of tests to make sure everything runs as expected ?
then we don't need the `dry-run` anymore